src/lib.rs

```diff
@@ -1014,6 +1015,13 @@ impl Plugin for EguiPlugin {
             "egui.wgsl",
             bevy_render::render_resource::Shader::from_wgsl
         );
+
+        app.add_systems(
+            PostUpdate,
+            update_accessibility
```

Comment:
Let's add this system to the `EguiPostUpdateSet::PostProcessOutput` set, and configure the whole set to run `before(AccessibilitySystem::Update)`.